### PR TITLE
i fucked up

### PR DESCRIPTION
**Pull request by MkLHX**
Hi @tuxgasy,

after a while using my fork dolibarr image project i realize i fucked up.
When i trying first to use your image on my host, i don't know why and i going too fast before think about what the issue on my host.
So i go to source code on your repository and see we can edit exposed database container port.

So now i realize the database container network is internal only.

The issue i encountered on my host is not from your image. So my first pull request https://github.com/tuxgasy/docker-dolibarr/pull/41 is a mistake and don't have to exist.

So if you can revet this PR do it, if you can't this new PR revert my editions.

I ran update.sh, test.sh 13.0.0 7.4, containers was built and everything works well.

I think i'm too hurry to contribute on dolibarr project.

Next time i take more time to check everything on local env before submit PR.

I applologies.

### CODE REVIEW

**Comment by tuxgasy** [comment]
I think we can leave this option. It can be useful to other people.

**Comment by MkLHX** [comment]
Hello,
I understand but right now the .env variable doesn't change the mysql engine port. Just change container exposed (public) port and use the same in each mysql query so inside stack containers the port have to rest 3306.

So my contribution does not work properly, if somebody change the .env DOLI_HOST_PORT that change the mysql engine port in request and "connection will be refused".

File: docker-run.sh [hunk 1]
```diff
@@ -29,7 +29,7 @@ EOF
 \$dolibarr_main_document_root_alt='/var/www/html/custom';
 \$dolibarr_main_data_root='/var/www/documents';
 \$dolibarr_main_db_host='${DOLI_DB_HOST}';
-\$dolibarr_main_db_port='${DOLI_DB_HOST_PORT}';
+\$dolibarr_main_db_port='3306';
```
Comment (mathieupotier) [inline docker-run.sh:32]: Better not hardcode this, DB is not part of the image and so you cannot be sure the implementation the client did on his infrastructure (changing standard port is a common way to secure your infra)...
It also could be a problem for a version compatible with other database engines such as Postgres.

File: docker-compose.yml [hunk 2]
```diff
@@ -17,6 +17,7 @@ networks:
 services:
     mariadb:
         image: mariadb:latest
+        env_file: .env
```
Comment (mathieupotier) [inline docker-compose.yml:20]: seems the `.env` file is missing from this commit.

File: docker-run.sh (hunk 1 above)
Comment (tuxgasy) [reply docker-run.sh:32]: I agree.

@MkLHX your first PR #41 is good. No need to revert it.

Comment (MkLHX) [reply docker-run.sh:32]: @mathieupotier ok i udenstand i just revert my edits.
the commit before my PR have this port harcoded.
https://github.com/tuxgasy/docker-dolibarr/blob/737234da339b16a40db762654147cda37c617617/docker-run.sh

File: docker-compose.yml (hunk 2 above)
Comment (MkLHX) [reply docker-compose.yml:20]: Sorry but for me the .env is never track by git

**Comment by mathieupotier** [comment]
I'm not sure that this PR is still relevant ...

**Comment by MkLHX** [comment]
so i can close ti

**Pull request closed without merging**